src/main/scala/common/memory.scala

```diff
@@ -67,12 +65,14 @@ class d2h2i1(val addrWidth : Int) extends Bundle{
    val clk = Input(Clock())
 }
 
-class AsyncReadMem(val addrWidth : Int) extends BlackBox{
+class AsyncReadMem(val addrWidth : Int) extends BlackBox with HasBlackBoxResource {
    val io = IO(new d2h2i1(addrWidth))
+   addResource("/vsrc/AsyncReadMem.sv")
```

Comment:
Can we switch these to be Chisel Mem and SyncReadMems?

Reply:
Done